templates/features.html

```diff
@@ -94,6 +94,12 @@
   search.value = location.hash.substr(1);
 }
 
+chromeMetadata.addEventListener('queryChanged', function(e) {
+  search.value = '=' + e.detail.version;
+  var event = new Event('input');
+  search.dispatchEvent(event);
```

Comment:
But this is inside a template. So I assume it shouldn't manipulate anything directly in the context it was put it to make things clear. So just assuming there is a global function to call might not be ideal. Or maybe you have something else in mind?